ml-agents/mlagents/trainers/learn.py

```diff
@@ -208,7 +208,7 @@ def run_training(
         options.docker_target_name,
         options.no_graphics,
         run_seed,
-        options.base_port + (sub_id * options.num_envs),
+        options.base_port + (sub_id * options.num_envs) + int(options.env_path!=None),
```

Comment:
Why the `+ int(options.env_path!=None)`?

Reply:
So the editor is port is by default different from the executable port

Reply:
Not particularly, since you can always set the executable port.  Is the motivation that you think the current setup is confusing to users?

In any case, since this is actually a conditional could we just handle it within an `if...else`?  I think this is more confusing than the equivalent which calls out the additional logic; would also leave space for a comment to explain.